### ISSUE TITLE
Add coverage support to verify command

### ISSUE THREAD

**Issue by dudicoco** [open-policy-agent/conftest#518]
Hi,

The `opa test` command has a `--coverage` flag which outputs the tests coverage.

Can this be added to `conftest verify`?

It would be great if it would be possible to make it exit with exit code 1 if coverage is not 100.

Thanks

**Comment by jpreese** [comment]
Not opposed to the idea of returning a non-zero error code based on coverage, but would lean more towards something along the lines of:

```
conftest verify ... --coverage=80
```

Where 80 represents the minimum percentage

**Comment by dudicoco** [comment]
@jpreese I did write in my first post that the `opa test` command has a `--coverage` flag, perhaps you missed it. :)

When do you think this feature could be added to `conftest` as well?

**Comment by jpreese** [comment]
Yep! I saw it, just leaving a note that they split coverage and threshold into two different flags, rather than just a single one.

At the moment, I don't have a timeline just whenever someone opens a pull request 😄

**Comment by dudicoco** [comment]
Thanks @jpreese.

Not a big deal, for now i'm using the opa binary for unit tests + coverage check.

**Comment by delamart** [comment]
Opa doesn't have the parse_config or parse_config_file functions so I can't do coverage when I use thoses.
